factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/internal/PgFactStore.java

```diff
@@ -80,20 +82,23 @@ public PgFactStore(
       @NonNull TokenStore tokenStore,
       @NonNull FactTableWriteLock lock,
       @NonNull FactTransformerService factTransformerService,
+      @NonNull PgFactIdToSerialMapper pgFactIdToSerialMapper,
       @NonNull PgSnapshotCache snapCache,
       @NonNull PgMetrics metrics) {
     super(tokenStore);
 
     this.jdbcTemplate = jdbcTemplate;
     this.subscriptionFactory = subscriptionFactory;
     this.lock = lock;
+    this.pgFactIdToSerialMapper = pgFactIdToSerialMapper;
     this.snapCache = snapCache;
     this.metrics = metrics;
     this.factTransformerService = factTransformerService;
   }
 
   @Override
   public @NonNull Optional<Fact> fetchById(@NonNull UUID id) {
+    // TODO lets see if this approach is the fastest
```

Comment:
It was already changed in another PR wasnt it? 🤔

Reply:
yes, but not on master yet